src/mixins/filing-template-mixin.ts

```diff
@@ -366,7 +366,8 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
           nrNumber: this.getNameRequest.nrNumber
         },
         parties: this.getAddPeopleAndRoleStep.orgPeople,
-        startDate: this.getRegistration.startDate
+        startDate: this.getRegistration.startDate,
+        businessTypeConfirm: this.getRegistration.businessTypeConfirm
```

Comment:
businessTypeConfirm is now getting the value from getRegistration in the frontend, but the backend isn't available yet.

Reply:
Draft registration filings are currently supported by the BE.

The BE will only parse the filing JSON when it is filed, not when it is still a draft. In both cases it will simply ignore businessTypeConfirm. So this field is saved to the db and retrieved from the db, but only the UI will use it.